### PR TITLE
Move overlay url

### PR DESCRIPTION
**Pull request by AkalUstat**
Screen Shot 2020-01-23 at 6 16 10 PM

### CODE REVIEW

**Comment by AkalUstat** [comment]
fixes #916

**Comment by maneetpaul** [comment]
@AkalUstat Can we add a little padding between the box and the URL so they arent so close together? 
Great job!

**Comment by AkalUstat** [comment]
@maneetpaul bhai sahib ji i made some styling changes directly from gh web client so im not sure if  the css worked. If you could check that would be nice 🙏

**Comment by AkalUstat** [comment]
Screen Shot 2020-01-24 at 4 03 04 PM
@saintsoldierx @maneetpaul

**Comment by AkalUstat** [comment]
@maneetpaul @saintsoldierx 
Screen Shot 2020-01-25 at 6 37 45 PM

**Comment by maneetpaul** [comment]
LGTM. A little padding on top and bottom as well would be nice but this works. 

@tsingh777 for code review

File: www/src/scss/_overlay.scss [hunk 1]
```diff
@@ -61,6 +61,30 @@
     width: auto;
   }
 }
+.url-container {
+  display: inline-block;
```
Comment (tsingh777) [inline www/src/scss/_overlay.scss]: not needed.

File: www/src/scss/_overlay.scss [hunk 2]
```diff
@@ -61,6 +61,30 @@
...
+  margin-top: 1vh;
+  padding: 8px;
+
+  & > div {
+    display: inline-block;
+  }
+  .input-wrap {
+    align-items: center;
```
Comment (tsingh777) [inline www/src/scss/_overlay.scss]: This is for `display: flex` you don't need it if your using inline-block everywhere. please remove.

File: www/src/scss/_overlay.scss [hunk 3]
```diff
@@ -61,6 +61,30 @@
     width: auto;
   }
 }
+.url-container {
+  display: inline-block;
+  margin-top: 1vh;
+  padding: 8px;
```
Comment (tsingh777) [inline www/src/scss/_overlay.scss]: this element should have either padding or margin (preferably margin). Let stick with one, use `margin: 9px` maybe?

File: www/js/overlay.js [hunk 4]
```diff
@@ -278,12 +266,28 @@ const toggleAnnouncements = h(
   h('div.setting-label', 'Announcement'),
 );
 
+const urlText = h(
+  'div.copy-text',
+  h('text.url', {
+    innerText: `${url}`,
```
Comment (tsingh777) [inline www/js/overlay.js]: when overlay is initially off we get this:
image
When it gets turned on the URL doesn't update and it still has `undefined` as the port. 

Can you add an update to the url when overlay is turned on. Also when it is off, maybe hide it?

**Pull request closed without merging**